tile.go

```diff
@@ -20,5 +20,21 @@ type Tile struct {
 
 // String formats a TyleType as a string. It returns the enum name.
 func (t TileType) String() string {
-	return "TODO format Tile"
+	// THIS IS THE METHODS
+	switch t {
+	case Wall:
+		return "Wall"
+
+	case Road:
+		return "Road"
+
+	case Exit:
+		return "Exit"
+
+	case Duplicate:
+		return "Duplicate"
+
+	default:
+		return "error,wrong input"
```

Comment:
Should we throw something like `NotImplementedException`

Reply:
In golang there are no "exceptions" error handling is done with the error interface.